src/scripts.js

```diff
@@ -133,35 +145,40 @@ function favoriteCard(event) {
 }
 
 function checkFavoriteActive() {
-  if (user.favoriteRecipes.length) {
+  if (!user.favoriteRecipes.length) {
+    return
+  } else {
     user.favoriteRecipes.forEach(recipe => {
       document.querySelector(`.favorite${recipe.id}`).classList.add('favorite-active')
     })
-  } else return
+  }
 }
 
 // function checkRecipeToCookActive() {
-//   if (user.recipesToCook.length) {
+//   if (!user.recipesToCook.length) {
+//     return
+//   } else {
 //     user.recipesToCook.forEach(recipe => {
-//       document.querySelector(`.to-cook${recipe.id}`).classList.add('to-cook-active')
+//       document.querySelector(`.to-cook${recipe.id}`).classList.add('to-cook-active');
 //     })
-//   } else return
+//   }
 // }
```

Comment:
We will need to fix this bug to make the recipe to cook to show it is active when filtering recipe to cook.